src/source/campaign-targeting.rst

```diff
@@ -0,0 +1,55 @@
+.. _campaign_targeting_overview:
+
+Campaign Targeting
```

Comment:
re-state here that this is still experimental, as you said in campaign-restrictions

Reply:
actually, after all the AMNext/XL merging, are we still in an experimental state? I think we can even release this part as stable?

Reply:
not for all tenants - I don't think all tenants supply buyer location data...